presto-postgresql/src/test/java/com/facebook/presto/plugin/postgresql/TestPostgreSqlPlugin.java

```diff
@@ -29,6 +29,6 @@ public void testCreateConnector()
     {
         Plugin plugin = new PostgreSqlPlugin();
         ConnectorFactory factory = getOnlyElement(plugin.getConnectorFactories());
-        factory.create("test", ImmutableMap.of("connection-url", "test"), new TestingConnectorContext());
+        factory.create("test", ImmutableMap.of("connection-url", "test", "connection-load-table-mappings", "false"), new TestingConnectorContext());
```

Comment:
So I'm not sure what you mean for this, I do have this tests updated/fixed in `sqlserver` module, as this particular one just seems to be to test that the plugin factory can be fetched and loaded and to create an instance of the plugin successfully.

I do see where defining the tests in `presto-base-jdbc` for handling mixed-case schema/table names that each JDBC implementation can pull to test individually in their own module if the DB handles mixed-case is useful.